extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/transferhistory/TransferHistoryPageApiService.java

```diff
@@ -91,7 +93,7 @@ public List<TransferHistoryEntry> getTransferHistoryEntries() {
             transferHistoryEntry.setState(transferProcessStateService.buildTransferProcessState(process.getState()));
             transferHistoryEntry.setTransferProcessId(process.getId());
             return transferHistoryEntry;
-        }).sorted(Comparator.comparing(TransferHistoryEntry::getLastUpdatedDate).reversed()).toList();
+        }).sorted(Comparator.comparing(TransferHistoryEntry::getLastUpdatedDate)).toList();
```

Comment:
are you sure you want to have newer transfer processes at the bottom of the list?